src/dbg/debugger.cpp

```diff
@@ -1079,6 +1098,8 @@ static void cbCreateProcess(CREATE_PROCESS_DEBUG_INFO* CreateProcessInfo)
             sprintf_s(command, "bp " fhex ",\"entry breakpoint\",ss", (duint)CreateProcessInfo->lpStartAddress);
             cmddirectexec(command);
         }
+
+        bTraceRecordEnabledDuringTrace = settingboolget("Engine", "TraceRecordEnabledDuringTrace");

```

Comment:
coz DBG_SETTINGS_UPDATED is not triggered when the application initializes so the `bTraceRecordEnabledDuringTrace` is not initialized. This line of code initializes it.